routes/favicon.coffee

```diff
@@ -0,0 +1,16 @@
+request = require 'request'
+
+url = null
+
+module.exports = (req, res, next) ->
+	if url?
+		res.redirect 301, url
+		return this
```

Comment:
Sorry for the delay in this.

What actually *is* `this` in this context? Is it anything useful?

What I meant in my other comment by "return this or return nothing on the next line" wasn't literal `this` but by "this" I meant "this which I am commenting on", particularly "whatever is returned by `res.redirect 301, url`; return that". (And what I meant by "nothing" was `undefined`, i.e. just a plain `return` statement.)

Justify `this` and I'll merge, otherwise let's have a plain `return` statement.